ingestion/setup.py

```diff
@@ -187,7 +189,7 @@ def get_long_description():
                 *[
                     requirements
                     for plugin, requirements in plugins.items()
-                    if plugin != "db2"
+                    if plugin not in {"airflow-container-1.10.15", "db2"}
```

Comment:
we need to move this here to now clash with the requests requirements from airflow 2